library/data/registry_entry_attachments/src/main/java/org/quiltmc/qsl/registry/attachment/api/DispatchedType.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 QuiltMC
+ * Copyright 2021-2022 QuiltMC
```

Comment:
ugh. can we fix the line ending bug in the licenser

Reply:
this isnt the line ending bug...

Reply:
This is actually normal and should have been done earlier.

The issue tho is that some files got the 2021 entirely replaced, and I am not able to reproduce that :/